src/drawer.js

```diff
@@ -288,6 +288,22 @@ $.Drawer.prototype = /** @lends OpenSeadragon.Drawer.prototype */{
     },
 
     // private
+    drawPlaceholder: function(rect, fillStyle) {
+        if (!this.useCanvas) {
+            return;
+        }
+
+        if ( typeof fillStyle === "function" ) {
+            this.context.fillStyle = fillStyle(this.context);
```

Comment:
What's the purpose of passing the context into the function? 

I wonder if there are other things worth passing into the function. For instance, we could pass in the TiledImage (in which case this line would probably happen in tiledimage.js), in case someone used the same function for multiple TiledImages but still wanted to differentiate between them. 

What do you think? Anything else?